src/hooks/useAccountRiskCheck.ts

```diff
@@ -35,8 +35,7 @@ export default function useAccountRiskCheck(account: string | null | undefined)
             .catch(() => dispatch(setOpenModal(null)))
         }
       } finally {
-        // leaving this code in place w/ a negligible cache time in case we want to increase cache time later
-        localStorage.setItem(riskCheckLocalStorageKey, (now + ms`10 seconds`).toString())
+        localStorage.setItem(riskCheckLocalStorageKey, (now + ms`1 day`).toString())
```

Comment:
This means it will take at least a day for any new blocks to take effect? Is this desirable?